install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -829,6 +829,13 @@ ciliumEndpointSlice:
   # fcfs groups together CiliumEndpoints in a first-come-first-serve basis, filling in the largest non-full slice first.
   sliceMode: identity
 
+# @schema
+# enum: ["agent", "operator", "both"]
+# @schema
+# -- Control whether CiliumIdentities are created by the agent ("agent"), the operator ("operator") or both ("both").
+# "Both" should be used only to migrate between "agent" and "operator".
```

Comment:
What's the "stability" of `both`-mode? Do we feel confident in exposing it to end-users, or should there be some sort of beta-level warning?

And can I tell users to call you when it breaks :-p?

Reply:
> Do we feel confident in exposing it to end-users, or should there be some sort of beta-level warning?

That's a good call, I'll add a beta level warning here.

Part of the motivation for this change is to make it easier to run E2E tests for each of these configurations (`operator` and `both` aren't covered at all right now), which is a prereq for graduating this feature.

Reply:
updated. FYI @dlapcevic @ovidiutirla this adds a line in the docs "Operator-managed identities is a beta feature"